README.md

```diff
@@ -73,6 +73,24 @@ our [blog post](https://blog.thunderbird.net/2023/07/k-9-mail-collaborates-with-
 
 You can report a security vulnerability [through the respective issues form](https://github.com/thunderbird/thunderbird-android/security/advisories/new).
 
+### Verifying the Apk Signature
+
+Users can ensure that the downloaded apk from Github and F-Droid was properly signed with our signing key by verifying
+its SHA-256 hash against the following values:
+
+- Thunderbird: `B6:52:47:79:B3:DB:BC:5A:C1:7A:5A:C2:71:DD:B2:9D:CF:BF:72:35:78:C2:38:E0:3C:3C:21:78:11:35:6D:D1`
+- Thunderbird Beta: `05:6B:FA:FB:45:02:49:50:2F:D9:22:62:28:70:4C:25:29:E1:B8:22:DA:06:76:0D:47:A8:5C:95:57:74:1F:BD`
+- K-9 Mail: `55:C8:A5:23:B9:73:35:F5:BF:60:DF:E8:A9:F3:E1:DD:E7:44:51:6D:93:57:E8:0A:92:5B:7B:22:E4:F5:55:24`
+
+To verify the SHA-256 hash of the APK, use the following command in your terminal:
+
+```bash
+keytool -printcert -jarfile <path-to-apk>
```

Comment:
I believe `keytool` will only verify the v1 signature. We probably want to recommend using `apksigner verify` so we don't have to update this in the future.

Reply:
The problem is that `apksigner` requires to install the Android SDK Build Tools.

Reply:
:shrug: We don't expect regular users to verify the APK signature. I think it's fine to require interested users to install the build tools. It's also possible they don't have `keytool` installed. So that would require some work as well.

Reply:
Let's go with apksigner, this is for example also what Signal does: https://signal.org/android/apk/